frontend/src/metabase/query_builder/components/dataref/QuestionPane/QuestionPane.tsx

```diff
@@ -75,7 +76,8 @@ const QuestionPane = ({
         <QuestionPaneDetail>
           <QuestionPaneIcon name="all" />
           <QuestionPaneDetailText>
-            {question.card().collection?.name || t`Our analytics`}
```

Comment:
The type error this commit fixes was on this line and read:
`Property 'collection' does not exist on type 'Card<DatasetQuery>'.`